addon.py

```
@@ -128,6 +128,46 @@ def _handle_client(self, client):
+                return None
+            
+            is_processing[0] = True
+            cmd = command_queue.pop(0)
+            
+            try:
+                # Execute command in Blender's main thread
+                response = self.execute_command(cmd)
+                response_json = json.dumps(response)
+                try:
+                    client.sendall(response_json.encode('utf-8'))
+                except:
```

Ruff (0.15.6)

[error] Do not use bare `except`

(E722)